### PR TITLE
JCLOUDS-1254 Filters storage account by current available locations

### PR DESCRIPTION
**Pull request by danielestevez**

### CODE REVIEW

File: azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/compute/AzureComputeServiceAdapter.java [hunk 1]
```diff
@@ -16,10 +16,23 @@
...
+import static com.google.common.collect.Iterables.find;
+import static com.google.common.collect.Iterables.getOnlyElement;
+import static org.jclouds.azurecompute.arm.compute.extensions.AzureComputeImageExtension.CONTAINER_NAME;
+import static org.jclouds.azurecompute.arm.compute.extensions.AzureComputeImageExtension.CUSTOM_IMAGE_OFFER;
+import static org.jclouds.azurecompute.arm.compute.functions.VMImageToImage.decodeFieldsFromUniqueId;
+import static org.jclouds.azurecompute.arm.compute.functions.VMImageToImage.encodeFieldsToUniqueIdCustom;
+import static org.jclouds.azurecompute.arm.compute.functions.VMImageToImage.getMarketplacePlanFromImageMetadata;
+import static org.jclouds.azurecompute.arm.config.AzureComputeProperties.IMAGE_PUBLISHERS;
+import static org.jclouds.compute.util.ComputeServiceUtils.metadataAndTagsAsCommaDelimitedValue;
+import static org.jclouds.util.Closeables2.closeQuietly;
```
Comment (danielestevez) [inline azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/compute/AzureComputeServiceAdapter.java]: My IDE is configured to put all static imports first. Not sure if there is an established style convention for this issue in jclouds.

**Comment by nacx** [comment]
The build failed due to a connection timeout. Let's restart it: rebuild please

**Comment by nacx** [comment]
rebuild please

**Comment by nacx** [comment]
Pushed to [master](http://git-wip-us.apache.org/repos/asf/jclouds-labs/commit/76353e75) and [2.0.x](http://git-wip-us.apache.org/repos/asf/jclouds-labs/commit/7214dcb4). Thanks @danielestevez!

**Pull request closed without merging**